dev/reference/alert-rules.md

```diff
@@ -0,0 +1,1423 @@
+---
```

Comment:
What does dev mean?

Reply:
master (the latest version)

Reply:
Do the changes above also apply to dev? @overvenus

Reply:
Yes, they do.

Reply:
Already updated. PTAL again. @overvenus